pkg/services/featuremgmt/registry.go

```diff
@@ -85,6 +85,12 @@ var (
 			Stage:       FeatureStagePublicPreview,
 			Owner:       grafanaExploreSquad,
 		},
+		{
+			Name:        "exploreContentOutline",
+			Description: "Content outline sidebar",
+			Stage:       FeatureStagePrivatePreview,
+			Owner:       grafanaExploreSquad,
+		},
```

Comment:
it looks like we can enable by default for only `FeatureStageGeneralAvailability` or `FeatureStageDeprecated` otherwise the test fails. i've updated it to use `FeatureStageGeneralAvailability`, but let me know if I need to change it.